### PR TITLE
Add the "Multiuser scenario" example

### PR DESCRIPTION
**Pull request by AndreyBelym**
Originally created by @felis2803. Unfortunately, I force-pushed some inappropriate commits (that wasn't connected to this repo, only to the [standalone example repo](https://github.com/DevExpress/testcafe-example-multiuser-scenario)) to his pull request, and it was closed automatically by GitHub, and it seems now I cannot commit anymore there.

I guess we still have to rewrite README according to the [template](https://github.com/DevExpress/testcafe-examples/pull/32).

### CODE REVIEW

**Review by felis2803 — APPROVED** [review]

File: detached-examples/multiuser-scenario/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Multiuser Scenario
+
+This example shows you how to start two TestCafe Runners and synchorize them to achieve testing with two different users in different browsers
```
Comment (titerman) [inline detached-examples/multiuser-scenario/README.md:3]:
```suggestion
You can not sign multiple users into the same website with a single instance of TestCafe. The solution to this limitation is to run multiple instances of TestCafe. This example shows you how to run multi-user tests with two or more synchronized TestCafe Runners.
```

**Review by titerman — CHANGES_REQUESTED** [review]

**Review by titerman — APPROVED** [review]

**Comment by miherlosev** [comment]
It's necessary adding this example link to the [main page](https://github.com/DevExpress/testcafe-examples#examples-in-this-repository) and the [documentation](https://testcafe.io/documentation/402637/examples) site.

I don't remember our decision about the location of the documentation issues.
Should we write an appropriate issue on GitHub or GitLab?
@AndreyBelym  @titerman

**Comment by titerman** [comment]
I still use GitHub to keep track of documentation-related issues, and plan on doing so going forward. Feel free to add an issue to the main TestCafe repo

**Comment by miherlosev** [comment]
Done - https://github.com/DevExpress/testcafe/issues/6213

**Pull request merged**